test/ts-node-unit-tests/test-utils.ts

```diff
@@ -27,8 +27,8 @@ export function reportError(error: (AssertionError & Error)): void {
     failure(`${code} ${errorMessage}
         ${stack?.split('\n')[1]}
 
-Got: ${printArrayOrString(actual as string | [])}
+Got: ${printArrayOrString(actual as any)}
```

Comment:
it is the same as in master. I changed it back and forth again, so please make sure you are looking at the diff with master, not since you checked last time :)